eahub/templates/base.html

```diff
@@ -137,6 +137,25 @@ <h4> /{{ page_name }}</h4>
     {% endblock %}
 
     <main>
+
+      <br>
```

Comment:
I wonder whether this could be removed? this tag isn't used for spacing anymore :)